packages/block-editor/src/components/block-popover/index.js

```diff
@@ -77,30 +77,36 @@ function BlockPopover(
 		};
 	}, [ selectedElement ] );
 
-	const { isZoomOut, sectionRootClientId, getParentSectionBlock } = useSelect(
-		( select ) => {
-			const {
-				isZoomOut: isZoomOutSelector,
-				getSectionRootClientId,
-				getParentSectionBlock: getParentSectionBlockFn,
-			} = unlock( select( blockEditorStore ) );
-
-			const root = getSectionRootClientId();
-			return {
-				sectionRootClientId: root,
-				isZoomOut: isZoomOutSelector(),
-				getParentSectionBlock: getParentSectionBlockFn,
-			};
-		},
-		[]
-	);
+	const {
+		isZoomOut,
+		sectionRootClientId,
+		getParentSectionBlock,
+		getBlockOrder,
+	} = useSelect( ( select ) => {
+		const {
+			isZoomOut: isZoomOutSelector,
+			getSectionRootClientId,
+			getParentSectionBlock: getParentSectionBlockFn,
+			getBlockOrder: getBlockOrderFn,
+		} = unlock( select( blockEditorStore ) );
+
+		const root = getSectionRootClientId();
+		return {
+			sectionRootClientId: root,
```

Comment:
Why do we have a variable here? Can't we follow the same pattern of calling the selector in the returned object?